packages/sveltekit/src/client/load.ts

```diff
@@ -1,11 +1,11 @@
+import { addTracingHeadersToFetchRequest } from '@sentry-internal/tracing';
```

Comment:
I'm not sure we can. `addTracingHeadersToFetchRequest` is needed in `BrowserTracing` and I can't move this function to `utils` (see https://github.com/getsentry/sentry-javascript/pull/7626#discussion_r1149455060). Also, moving it to the browser package wouldn't work because then we'd get a circular dependency between tracing-internal and browser. 

Def open to other ideas though 😅

Reply:
Alright then no worries, we keep it in `tracing-internal` for now then.